aws/logs_monitoring/lambda_function.py

```diff
@@ -757,13 +786,22 @@ def normalize_events(events, metadata):
     for event in events:
         events_counter += 1
         if isinstance(event, dict):
-            normalized.append(merge_dicts(event, metadata))
+            normalized_event = merge_dicts(event, metadata)
```

Comment:
Why do you drop the `append`? Won't this break everything?

Reply:
Append is moved to line 803, instead of having many possible places in which we append to `normalized` it's done only once at the end of the loop